src/TreeContainer.js

```diff
@@ -19,57 +19,63 @@ import {
     omit,
     pick,
     propOr,
-    type
```

Comment:
Oh interesting - seems the pattern we have in our `prettier` calls in the `package.json` miss the top level files in `src/` - would you mind changing
`src/**/*.js src/**/*.react.js`
to
`src/*.js src/**/*.js`
so we know this formatting is consistent with the rest of our codebase?
(the `*.react.js` variant also doesn't seem to be necessary, `*.js` covers it)